main.py

```diff
@@ -307,7 +308,29 @@ def analyze_submission(submission):
     elif giveaway_regex.search(title):
         alert_interested_users('ga', giveaway_regex.search(title).group('title'), submission)
     print(f"Finished regex search and alerts in {round(time.time()-t, 1)} seconds", flush=True)
-    
+
+# Comparison function for ordering commands with /rm {index} commands coming
+# first in descending order by index.
+# Other commands (e.g. /h) can be in any order. This is used to support the
+# removal of multiple items in the same message.
+def _compare_by_rm_index(command1, command2):
+    if command1[0] != '/rm' and command2[0] != '/rm':
+        return 0
+    if command1[0] == '/rm' and command2[0] != '/rm':
+        return -1
+    if command1[0] != '/rm' and command2[0] == '/rm':
+        return 1
+
+    rm_item_1 = command1[1].strip()
+    rm_item_2 = command2[1].strip()
+
+    if rm_item_1.isdigit() and not rm_item_2.isdigit():
+        return -1
+    if not rm_item_1.isdigit() and rm_item_2.isdigit():
+        return 1
+    if rm_item_1.isdigit() and rm_item_2.isdigit():
+        return 1 if rm_item_1 < rm_item_2 else -1
```

Comment:
Convert to int before comparison
```suggestion
        return 1 if int(rm_item_1) < int(rm_item_2) else -1
```

Reply:
Good catch, thanks!